lib/features/create_transaction/repository/data_providers/create_transaction_bff_backend_data_provider.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:developer';
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+import 'package:givt_app_kids/features/create_transaction/models/transaction.dart';
+
+class CreateTransactionBffBackendDataProvider {
+  Future<void> createTransaction({required Transaction transaction}) async {
+    const hardcodedToken =
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1laWQiOiI0NkRBNTU5RC1CQ0M4LTQ0RjgtQjVFOC0wNzQwRjMxNEQzODMiLCJ1bmlxdWVfbmFtZSI6ImdpdnQtdGVhbUBnaXZ0YXBwLm5ldCIsInJvbGUiOiJnaXZ0T3BlcmF0b3IiLCJBcHBJZCI6IkludGVybmFsIGRhc2hib2FyZCAoUmV0b29sKSIsIm5iZiI6MTY4MTgxMDY4OSwiZXhwIjoxOTIxODEwNjg5LCJpYXQiOjE2ODE4MTA2ODksImlzcyI6Imh0dHBzOi8vYXBpLmdpdnRhcHAubmV0In0.y4QL_WZcPGwpmD471RmnMwFtMFlNYvzcpbiblqgHkBw';
+    try {
```

Comment:
Why don't you use an interceptor to add the token to the request?

Reply:
The interceptor is in our backlog, but moved to the next sprint